.github/workflows/deploy.yml

```
@@ -0,0 +1,45 @@
+      - name: SSH into Server and Deploy
+        run: |
+          expect <<EOF
+          spawn ssh -o StrictHostKeyChecking=no -p ${{ secrets.SSH_PORT }} ${{ secrets.SSH_USER }}@${{ secrets.SSH_HOST }}
+          expect "password:"
+          send "${{ secrets.SSH_PASSWORD }}\r"
+          expect "$ "
+          send "cd /home/${{ secrets.SSH_USER }}/public_html/hmvc-laravel/\r"
+          send "git pull origin main\r"
+          send "composer2 install --no-interaction --prefer-dist --optimize-autoloader\r"
+          send "composer2 update --no-interaction\r"
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)